### PR TITLE
Update documentation for BUILD.md, INSTALL.md, and add requirements.txt

### PR DESCRIPTION
**Pull request by sspintel**

### CODE REVIEW

File: docs/BUILD.md [hunk 1]
```diff
@@ -1,84 +1,87 @@
-# Installation and Build Options
-## Prerequisites
+# Build Instructions
 
-|Build Type| Requirements|
-|:-----------------------|-------------------|
-|Use pre-built packages| Python 3.6, 3.7, or 3.8, TensorFlow v2.5.0|
-|Build from source| Python 3.6, 3.7, or 3.8, GCC 7.5 (Ubuntu 18.04, 20.04),  cmake 3.14 or higher, Bazelisk v1.7.5, virtualenv 16.0.0 or higher, patchelf 0.9, libusb 1.0.0|
+<!-- vscode-markdown-toc -->
+* 1. [Prerequisites](#Prerequisites)
```
Comment (rrajore) [inline docs/BUILD.md:4]: Extra * can be removed. This is adding extra bullet heading

File: docs/INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        pip3 install pip==21.0.1
+        pip3 install tensorflow==2.5.0
+        pip3 install openvino-tensorflow
+<br/> 
+
+##  1.2. <a name='InstallOpenVINOintegrationwithTensorFlowalongsidetheIntelDistributionofOpenVINOToolkit'></a>Install **OpenVINO™ integration with TensorFlow** alongside the Intel® Distribution of OpenVINO™ Toolkit
+
+This **OpenVINO™ integration with TensorFlow** package is currently compatible with OpenVINO™ version 2021.4. This package supports Intel<sup>®</sup> CPUs, Intel<sup>®</sup> integrated GPUs, Intel<sup>®</sup> Movidius™ Vision Processing Units (VPUs), and Intel<sup>®</sup> Vision Accelerator Design with Movidius™ (VAD-M).
+
+You can build [TensorFlow from source](https://github.com/openvinotoolkit/openvino_tensorflow/blob/master/docs/BUILD.md#tensorflow) with -D_GLIBCXX_USE_CXX11_ABI=1 or use the following TensorFlow package:
```
Comment (rrajore) [inline docs/INSTALL.md:36]: Last part should be " or follow below instructions to install appropriate Tensorflow"

**Review by rrajore — COMMENTED** [review]
Have added some comments. Please see if they can be addressed

File: docs/BUILD.md (hunk 1 above)
Comment (ck-intel) [reply docs/BUILD.md:4]: Auto generated using a vscode extensions, will manually modify it.

File: docs/BUILD.md [hunk 3]
```diff
@@ -1,84 +1,87 @@
...
+* 2. [OpenVINO™ integration with TensorFlow](#OpenVINOintegrationwithTensorFlow)
+	* 2.1. [Build Instructions](#BuildInstructions)
+	* 2.2. [Build Instructions for Intel Atom® Processor](#BuildInstructionsforIntelAtomProcessor)
+	* 2.3. [ Build Verification](#BuildVerification)
+* 3. [OpenVINO™](#OpenVINO)
+* 4. [TensorFlow](#TensorFlow)
+* 5. [Build ManyLinux2014 compatible **OpenVINO™ integration with TensorFlow** wheels](#BuildManyLinux2014compatibleOpenVINOintegrationwithTensorFlowwheels)
 
-## Use Pre-Built Packages
+<!-- vscode-markdown-toc-config
```
[the remark this answers is not in the document]
Comment (ck-intel) [reply docs/BUILD.md:15]: Auto generated using a vscode extensions, will manually modify it.

File: docs/BUILD.md [hunk 4]
```diff
@@ -90,45 +91,44 @@ $ git submodule init
 $ git submodule update --recursive
 ```
 
-Install the following python packages
+###  2.1. <a name='BuildInstructions'></a>Build Instructions
+Use one of the following build options based on the requirements. **OpenVINO™ integration with TensorFlow** built using PyPI TensorFlow enable only the Python APIs, TensorFlow C++ libraries built from source is required for using the C++ APIs.
```
Comment (rrajore) [inline docs/BUILD.md:95]: Correction at below places. This is corrected version
 - <TensorFlow enables only>
 -  <required to use C++ APIs

File: docs/BUILD.md [hunk 5]
```diff
@@ -90,45 +91,44 @@ $ git submodule init
...
 
-### **OpenVINO™ integration with TensorFlow**
-Use one of the following build options based on the requirements
-
-1. Pulls compatible prebuilt TF package from PyPi, clones and builds OpenVINO™ from source.
-
-        python3 build_ovtf.py
+        python3 build_ovtf.py --tf_version=v2.5.0 --openvino_version=2021.4
 
 2. Pulls compatible prebuilt TF package from PyPi. Uses OpenVINO™ binary.
```
Comment (rrajore) [inline docs/BUILD.md:101]: Write "Uses OpenVINO™ binary from specified location"

File: docs/INSTALL.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+<br/>  
+
+##  1.1. <a name='InstallOpenVINOintegrationwithTensorFlowalongsidePyPiTensorFlow'></a>Install **OpenVINO™ integration with TensorFlow** alongside PyPi TensorFlow
+
+This **OpenVINO™ integration with TensorFlow** package includes pre-built libraries of OpenVINO™ version 2021.4. The users do not have to install OpenVINO™ separately. This package supports Intel<sup>®</sup> CPUs, Intel<sup>®</sup> integrated GPUs, and Intel<sup>®</sup> Movidius™ Vision Processing Units (VPUs).
+
+
+        pip3 install pip==21.0.1
+        pip3 install tensorflow==2.5.0
+        pip3 install openvino-tensorflow
```
Comment (rrajore) [inline docs/INSTALL.md:29]: -U option needed?

File: README.md [hunk 7]
```diff
@@ -29,14 +29,14 @@ This **OpenVINO™ integration with TensorFlow** package comes with pre-built li
 - Intel<sup>®</sup> Movidius™ Vision Processing Units (VPUs)
 
 
-        pip3 install -U pip==21.0.1
-        pip3 install -U tensorflow==2.5.0
-        pip3 install openvino-tensorflow
+        pip3 install pip==21.0.1
+        pip3 install tensorflow==2.5.0
+        pip3 install -U openvino-tensorflow
```
[the remark this answers is not in the document]
Comment (ck-intel) [reply README.md:36]: Added in main.js file

**Review by rrajore — APPROVED** [review]
Changes look fine.

**Pull request merged**